rule_engine/src/main.cpp

```diff
@@ -52,13 +60,22 @@ namespace
 
 	auto genquery2_execute(std::list<boost::any>& _rule_arguments, irods::callback& _effect_handler) -> irods::error
 	{
-		log_rule_engine::info(__func__);
-		log_rule_engine::info("number of arguments = [{}]", _rule_arguments.size());
+#ifdef IRODS_ENABLE_42X_COMPATIBILITY
+		rodsLog(LOG_NOTICE, __func__);
+		rodsLog(LOG_NOTICE, fmt::format("Number of arguments = [{}]", _rule_arguments.size()).c_str());
```

Comment:
Should these log levels align with `trace` and `debug` respectively?

Reply:
Absolutely. Will adjust.

Reply:
Fixed.